src/Features/Core/Portable/ChangeSignature/IChangeSignatureOptionsService.cs

```diff
@@ -1,15 +1,17 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using Microsoft.CodeAnalysis.Host;
-using Microsoft.CodeAnalysis.Notification;
 
 namespace Microsoft.CodeAnalysis.ChangeSignature
 {
     internal interface IChangeSignatureOptionsService : IWorkspaceService
     {
         ChangeSignatureOptionsResult GetChangeSignatureOptions(
             ISymbol symbol,
+            int insertPosition,
             ParameterConfiguration parameters,
-            INotificationService notificationService);
+            Document document);
+
+        AddedParameterResult GetAddedParameter(Document document, int insertPosition);
```

Comment:
what does this mean?

Reply:
added a doc

---
In reply to: [367214712](https://github.com/dotnet/roslyn/pull/38748#discussion_r367214712) [](ancestors = 367214712)